lib/ansible/modules/windows/win_psmodule.ps1

```diff
@@ -9,18 +10,33 @@
 
 $params = Parse-Args $args -supports_check_mode $true
 
-$name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true
+$name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true 
```

Comment:
This PR added an extra space at the end of the line, are you able to remove it?

Reply:
Will do